### ISSUE TITLE
Missing security considerations section.

### ISSUE THREAD

**Issue by steely-glint** [fluffy/draft-jennings-game-state-over-rtp#22]
If you are going to define a new binary format in an RFC you will need a security considerations section.
The document should also describe why none of the existing, well studied, binary formats were found suitable.

**Comment by fluffy** [comment]
Makes sense. (and 100% agree on security section, just not done yet). Any thoughts on the existing binary formats I should compare to ?
